recipe/meta.yaml

```diff
@@ -20,12 +20,12 @@ requirements:
     - python
     - pip
     - ruamel
-    - setuptools >=20.6.8
+    - setuptools
     - wheel
   run:
     - python
     - setuptools
-    - ruamel.yaml.clib >=0.1.2
+    - ruamel.yaml.clib >=0.2.6
 
 test:
   imports:
```

Comment:
Could you add a description:
`A YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order`
see https://pypi.org/project/ruamel.yaml/